packages/mds-db/index.ts

```diff
@@ -1266,6 +1266,14 @@ async function readStatusChanges(
     }
   }
 
+  if (device_id) {
+    if (!isUUID(device_id)) {
```

Comment:
Real dumb question, but isn't the type declaration supposed to ensure this is a UUID?

Reply:
If I understand your question correctly, ... The `UUID` type is just an alias for `string`. Regardless though, static type checking only helps at compilation time. At run time, you have to type guard everything. That's why most of the validators I've written take an input argument of type `arg: unknown` and returns `arg is <type>`. Which is basically, accept this argument of unknown type, apply some tests/constraints to it, and it they all pass, treat it as a `<type>` from that point forward in the code.

Reply:
Ah right, that makes sense. Thanks.